server/seeds/travelData.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "id": "1", "category id": "3", "item name": "The Ednaria Package", "despription": "A weekend get away from all lifeforms to Abaroo with gorgeous views in a quaint coattage on 2 splorgons of land.", "price": "38,000 Zorpians" 
+    },
+    {
+        "id": "2", "category id": "3", "item name": "The Gatheron Up Package", "despription": "This is a trip specific for reunions! Get your families together in this giant 10 bedroom home for a week on the planet of Vogsphere. Full stocked fridge!!!", "price": "78,000 Zorpians" 
+    },
+    {
+        "id": "3", "category id": "3", "item name": "The Dexterio Package", "despription": "ROMANTIC GET AWAY FOR 2! Take this 2 night trip to Arda! 2 bedrooms, 1 bathroom You will sleep on a literal bed of roses! (Watch out for the thorns!) But it is as romantic as romance comes!", "price": "99,000 Zorpians" 
+    },
+    {
+        "id": "4", "category id": "3", "item name": "The Bornia Package", "despription": "", "price": "25,000 Zorpians" 
+    },
+    {
+        "id": "5", "category id": "3", "item name": "The Plaquid Package", "despription": "Trip for 2 all the way to and from Decapod 10! Get the real Decapodian experience with all you can eat fish and chess peices!", "price": "10,000 Zorpians" 
+    },
+    {
+        "id": "6", "category id": "3", "item name": "The Quintonias Package", "despription": "Trip to and from Dagoban for 2. Go visit this beautiful swamp-covered plaet and gain some extra wisdom!", "price": "" 
+    },
+    {
+        "id": "7", "category id": "3", "item name": "The Aebos Package", "despription": "Travel to the remanence of the planet Shlorp. See what was once a perfect utopia but is unfortunatley no more! Full tour!", "price": "500 Zorpians" 
+    },
+    {
+        "id": "8", "category id": "3", "item name": "The Planetaira Package", "despription": "A trip for 2 adults and 2 children. You will spend 3 nights on Naboo. then you will be taken to a 2 night stay in a beautiful castle on Ego! This pricec includes your trip to and from home! (meals not included.)", "price": "1,000,000 Zorpians"" 
```

Comment:
I'm seeing an extra quotation mark at the end after "Zorpians", might need to change that right quick!